vaadin-text-field-flow-parent/vaadin-text-field-flow/src/main/java/com/vaadin/flow/component/textfield/EmailField.java

```diff
@@ -37,13 +37,55 @@
 
 /**
  * Email Field is an extension of Text Field that only accepts email addresses
- * as input. If the given address is invalid, the field is highlighted in red
- * and an error message appears underneath the input. The validity of the email
- * addresses is checked according to the RFC 5322 standard, which includes the
- * format for email addresses. The component also supports supplying additional
- * validation criteria using a regular expression (see
- * {@link #setPattern(String)}). These extra validation criteria can be used,
+ * as input.
+ *
+ * <h2>Validation</h2>
+ * <p>
+ * Email Field comes with a built-in validation mechanism based on constraints.
+ * Validation is triggered when the user applies an input change, for example,
+ * by pressing Enter, blurring the field, or when the value changes
+ * programmatically. In eager and lazy value change modes, validation is also
+ * triggered on every character press with a delay according to the selected
+ * mode.
```

Comment:
```suggestion
 * Validation is triggered when the user applies an input change, for example,
 * by pressing Enter or blurring the field. Programmatic value changes trigger
 * validation as well. In eager and lazy value change modes, validation is also
 * triggered on every character press with a delay according to the selected
 * mode.
```

Reply:
Yeah, that's better, although you could remove the comma after "for example".

Reply:
Applied.